### ISSUE TITLE
Bad translations string usage

### ISSUE THREAD

**Issue by sfan5** [minetest/minetest#11442]
https://github.com/minetest/minetest/blob/effb5356caee01c8bfa63e98974347aab01f96ef/src/gui/modalMenu.cpp#L265-L271

Piecing together translated fragments is a no-go. This should be a format string.

**Comment by Wuzzy2** [comment]
Yes. I actually tried to fix this recently, but I struggled with that ANDROID define (I don't have Android, and have no idea how to code for it, or how to build it). I have made similar fixes of string concatenation recently: https://github.com/minetest/minetest/commit/51bf4a6e26f9eca461ae88181b06b517afc4d656

I believe the fix for this will look very similar.

According to my analyis, this also seems to be the final instance of string concatenation in the code, the others have been fixed.

Your issue is technically a duplicate of #5850, but as you've given more detail, I am closing #5850 now.
